include/reelmagic.h

```diff
@@ -38,16 +38,15 @@ struct ReelMagic_VideoMixerMPEGProvider {
 
 void ReelMagic_RENDER_SetPal(uint8_t entry, uint8_t red, uint8_t green, uint8_t blue);
 
-void ReelMagic_RENDER_SetSize(const uint16_t width, const uint16_t height,
+void ReelMagic_RENDER_SetSize(const uint32_t width, const uint32_t height,
                               const bool double_width, const bool double_height,
                               const Fraction& pixel_aspect_ratio,
-                              const uint8_t bits_per_pixel,
+                              const uint32_t bits_per_pixel,
```

Comment:
Same here. Not sure why you needed to change it back from uint8_t. I have changed bpp to uint8_t in most places and I'm finishing the job in my big VGA related PR currently under review.

Reply:
Separate commit please. I'm gonna change these back to uint8_t when my PR gets merged, but whatever.